docs/articles/documentation/how-it-works/README.md

```diff
@@ -3,151 +3,40 @@ layout: docs
 title: How It Works
 permalink: /documentation/how-it-works/
 ---
-# How It Works
+# How Does TestCafe Work?
 
-Before TestCafe, end-to-end web testing tools that worked with real browsers relied on driving these browsers from the outside to emulate user actions. This approach, however, had serious drawbacks.
+The automation engine behind TestCafe takes over browsers and the web pages they display to simulate user activity.
 
-* **Dependency on third-party tools.** This required you to install, configure, and maintain additional drivers, SDKs and/or libraries in every test environment.
-* **Limited browser support.** The range of available browsers and platforms was restricted to those supported by the driver.
-* **Boilerplate code.** These testing tools functioned with the same browser instance during the entire test run, so the tested web app state (cookies, storages, etc.) persisted from test to test. As a result, end-to-end tests contain a large amount of boilerplate code that clears the state to avoid interference between tests.
-* **Low automation.** Legacy tools were unable to interact directly with the tested page. Although the fact that the user’s test code did not intervene in the app code was an advantage, the testing tool itself could benefit from operating inside the page. This would enable the tool to automate many routine tasks.
+## The Server-Client Architecture
 
-## TestCafe is Different
+The server-client architecture of TestCafe makes it capable of executing both system-level and in-browser code.
```

Comment:
Are we sure it's `system-level` code, not `standalone` or smth? I'm not 100% sure what `system-level code` means.

Like:
*makes it capable of executing both standalone and in-browser code.*
*makes it capable of executing code both in a standalone process and in the browser.*